apps/dashboard/lib/smart_attributes/attribute.rb

```diff
@@ -31,16 +31,28 @@ def value
       end
     end
 
+    # Check select widget has options values provided
+    # @return [StandardError] if missing any values
+    def validate!
+      if widget == 'select' && (select_choices.size != select_choices.compact.size)
+        raise StandardError, I18n.t('dashboard.batch_connect_form_invalid', id: id)
+      end
+
+      self
+    end
```

Comment:
I think this is a part of #1455 which I don't believe we want to backport either.